core/src/main/java/tech/tablesaw/columns/numbers/IntParser.java

```diff
@@ -56,10 +52,13 @@ public int parseInt(String str) {
     if (isMissing(str)) {
       return IntColumnType.missingValueIndicator();
     }
-    String s = str;
+    return Integer.parseInt(normalize(str));
+  }
+
+  private String normalize(String str) {
     if (ignoreZeroDecimal) {
-      s = StringUtils.removeZeroDecimal(s);
+      str = StringUtils.removeZeroDecimal(str);
```

Comment:
reassigning a variable causes a warning in codacy

Reply:
Yep I noticed. I was thinking the code was like this to begin with and it is easy to end up with a growing task when starting to fix issues not introduced by the current PR. It is a also questionable IMHO that reassigning variables is bad practice. I am happy to change it.